pootle/templates/editor/units/edit.html

```diff
@@ -217,10 +217,10 @@
               <time
                 title="{{ unit.change.submitted_on|dateformat }}"
                 datetime="{{ unit.change.submitted_on.isoformat }}">{{ unit.change.submitted_on|relative_datetime_format }}</time>
-              {% with reviewer=unit.change.reviewed_by %}
-              {% if reviewer %}
+              {% with accepted_suggestion=latest_target_submission.suggestion %}
+              {% if accepted_suggestion %}
               <br>
-              <span>({% blocktrans with profile_url=reviewer.get_absolute_url%}reviewed by <a href="{{ profile_url }}">{{ reviewer }}</a>{% endblocktrans %})</span>
+              <span>({% blocktrans with profile_url=accepted_suggestion.reviewer.get_absolute_url reviewer=accepted_suggestion.reviewer %}reviewed by <a href="{{ profile_url }}">{{ reviewer }}</a>{% endblocktrans %})</span>
```

Comment:
Nit: use `trimmed` in `blocktrans` to avoid having such long lines in templates. More in https://docs.djangoproject.com/en/1.11/topics/i18n/translation/#blocktrans-template-tag